pkgs/applications/misc/nwg-panel/default.nix

```diff
@@ -10,17 +10,18 @@
 , pamixer          # pamixer
 , pulseaudio       # pactl
 , libdbusmenu-gtk3 # tray
+, playerctl
 }:
 
 python3Packages.buildPythonApplication rec {
```

Comment:
please use `finalAttrs` idiom instead of `rec` to allow proper overriding (see https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes)